### DISCUSSION TITLE
Emotion Demo Sandbox Not Working

### DISCUSSION THREAD

**Discussion by youngsterjaidev** [emotion-js/emotion#2369] [Q&A]
![Annotation 2021-05-07 220836](https://user-images.githubusercontent.com/47059807/117481699-f53ca880-af80-11eb-8922-054d8fdadc04.jpg)

I try run this example in my system and also try the documentation but is not working ..... Somebody help me !!!!

**Comment by Andarist** [comment]
I let @tkh44 about this and this codesandbox will be fixed soon-ish.

**Reply by youngsterjaidev** [reply]
Thank you So Much Sir.....
